app/helpers/submissions_helper.rb

```diff
@@ -44,25 +44,38 @@ def set_release_on_results(groupings, release)
   end
 
   def get_submissions_table_info(assignment, groupings)
-    parts = groupings.select &:has_submission?
-    results = Result.where(submission_id:
-                             parts.map(&:current_submission_used))
+    if current_user.is_a_reviewer?(assignment)
+      
```

Comment:
It was an attempt to avoid the Result query when it's not needed, but looking at it I realize I could just flip the `is_a_reviewer` check to a "not" check and avoid the empty block.